packages/flutter/lib/src/painting/basic_types.dart

```diff
@@ -2,6 +2,13 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+/// @docImport 'package:flutter/rendering.dart';
+/// @docImport 'package:flutter/material.dart';
```

Comment:
Do you sort these? We don't'\ have a lint yet 😛

Reply:
I tried to manually sort these. Looks like messed up my ABCs here. Will fix - and cannot wait for the lint!